main/webapp/modules/core/scripts/views/data-table/menu-reconcile.js

```diff
@@ -140,7 +140,7 @@ DataTableColumnHeaderUI.extendMenu(function(column, columnHeaderUI, menu) {
     var frame = DialogSystem.createDialog();
     frame.width("400px");
 
-    var header = $('<div></div>').addClass("dialog-header").text($.i18n('core-views/use-values-as-identifiers')).appendTo(frame);
+    var header = $('<div></div>').addClass("dialog-header").text($.i18n('core-views/use-values-as-identifiers-note')).appendTo(frame);
```

Comment:
As mentioned above, I do not think we should be making any changes to this dialog in this pull request.

Reply:
The same has introduced a conflict with master. Am currently working on a different approach of displaying the message